recipes/libiconv/all/conanfile.py

```diff
@@ -15,7 +15,6 @@
 from conan.tools.layout import basic_layout
 from conan.tools.microsoft import is_msvc, unix_path
 from conan.tools.scm import Version
-from conans.tools import get_gnu_triplet
```

Comment:
why not use `get_gnu_triplet`? is it buggy or something?
repeating the same logic in every recipe doesn't scale

Reply:
It's not migrated to 2.0 yet 🙃

Reply:
let's create an issue to migrate it then.
every old v1 `tools.*` should have a drop-in replacement in v2, or at very least, some clear migration path how to replace it.
let's avoid adding workarounds that don't scale to recipes.